service.go

```diff
@@ -0,0 +1,63 @@
+package wallet
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/big"
+	"net/http"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+type TransactionRequest struct {
+	ChainID int64
+	To      string
+	Value   big.Int
+	Data    string
+}
+
+func ProcessRequest(w *Wallet, request TransactionRequest) (reply string, err error) {
```

Comment:
Are you going to rebuild the user's signature with other wallet? Why do we need to sign with another pv?

Reply:
The wallet will keep the private key, and the user has to enter a passphrase in order to use the private key to sign.